script.js

```diff
@@ -168,10 +168,16 @@ document.addEventListener('DOMContentLoaded', function() {
 });
 
 $( document ).ready(function() {
+
   $('a.submit-a-request').each(function () {
```

Comment:
didn't get why we are looping here?

Reply:
hiding all `submit-a-request` links over the page but it is old implementation.